Misc/NEWS.d/next/Library/2021-07-04-11-33-34.bpo-41249.sHdwBE.rst

```diff
@@ -0,0 +1,5 @@
+Fixes TypedDict to work with get_type_hints() and postponed evaluation of
+annotations across modules.
+
+This is done by adding the module name to ForwardRef at the time the object
+is created and using that to resolve the globals during the evaluation.
```

Comment:
Usually technical implementation details are left out of the news article. The commit log is enough for that. WDYT?
```suggestion
```